src/assets/wise5/vle/vle.component.html

```diff
@@ -26,7 +26,7 @@
 
 <ng-template #defaultVLETemplate>
   @if (layoutState === 'node') {
-    <step-tools class="control-bg-bg mat-elevation-z1" />
+    <step-tools class="control-bg-bg mat-elevation-z1" [timedStep]="isCurrentNodeTimed()" />
```

Comment:
Calculate timedStep boolean in  StepToolsComponent.updateModel()? Remove isCurrentNodeTimed() function?
```suggestion
    <step-tools class="control-bg-bg mat-elevation-z1" />
```